packages/documentation/src/stories/components/table/table.docs.mdx

```diff
@@ -25,3 +25,8 @@ import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 When buttons are placed in a table, be sure to use `alignment: middle`, respectively `<tbody class="align-middle">` to nicely align the content and the buttons.
 
 <Canvas of={TableStories.TableWithButtons} />
+
+### Interactive Table in Angular
+
+If an interactive data table is needed, the design system recommends the table from primeng. However, these can only be used for Angular. The documentation contains all the necessary information on how to use the primeng-table. The LTS versions would need a license, so it is recommended not to use LTS.
```

Comment:
```suggestion
If an interactive data table is needed, we recommend the table component from the PrimeNG UI Suite. To make it look like a Swiss Post component, please use our [@swisspost/design-system-styles-primeng](/?path=/docs/d2112bed-c611-4098-a1ad-e654f7d622e7--docs) package.

However, this component can only be used in Angular projects. Their [documentation](https://primeng.org/table) contains all the necessary information on how to use it.

The LTS versions would need a license, so it's recommended not to use LTS.
```

Reply:
@cedic3 The docs page for the PrimeNG table styles package is not yet merged/released, but among the latest PRs #2718